### PR TITLE
Use common core option when using the Lomap Atom Mapper

### PR DESCRIPTION
**Pull request by richardjgowers**
Gives speed up of over 10x

### CODE REVIEW

**Comment by richardjgowers** [comment]
I think there's a better way to do this eventually, but this is such a nice win that I'd rather take this victory today then do the better way in time

File: openfe/setup/atom_mapping/ligandatommapper.py [hunk 1]
```diff
@@ -14,8 +14,6 @@ class LigandAtomMapper(gufe.AtomMapper):
     Subclasses will typically implement the ``_mappings_generator`` method,
     which returns an iterable of :class:`.LigandAtomMapping` suggestions.
     """
-    _no_element_changes: bool = False #TODO: to be removed
```
Comment (IAlibay) [inline openfe/setup/atom_mapping/ligandatommapper.py:17]: Huh was this even ever used by anything?

**Review by IAlibay — APPROVED** [review]
LGTM, it's not exactly the least elegant thing in our stack tbh.

File: openfe/setup/atom_mapping/ligandatommapper.py (hunk 1 above)
Comment (richardjgowers) [reply openfe/setup/atom_mapping/ligandatommapper.py:17]: It's the stub of something that was already removed

**Pull request merged**